### PR TITLE
Fix error in AddApplicant usage message

### PR DESCRIPTION
**Pull request by seanleong339**

### CODE REVIEW

File: src/main/java/seedu/address/logic/commands/applicant/AddApplicant.java [hunk 1]
```diff
@@ -30,12 +30,12 @@
...
             + "Example: " + COMMAND_WORD + " "
             + PREFIX_NAME + "John Doe "
-            + PREFIX_NRIC + "S12345678D "
+            + PREFIX_NRIC + "S1234567D "
             + PREFIX_PHONE + "98765432 "
             + PREFIX_EMAIL + "johnd@example.com "
             + PREFIX_ADDRESS + "311, Clementi Ave 2, #02-25 "
-            + PREFIX_DATEAPPLIED + "21-3-2022 "
-            + PREFIX_TAG + "new applicant "
+            + PREFIX_DATEAPPLIED + "2022-3-21 "
```
Comment (e0543517) [inline src/main/java/seedu/address/logic/commands/applicant/AddApplicant.java:37]: The month of date applied should be 2022-03-21.

**Review by e0543517 — CHANGES_REQUESTED** [review]

File: src/main/java/seedu/address/logic/commands/applicant/AddApplicant.java (hunk 1 above)
Comment (seanleong339) [reply src/main/java/seedu/address/logic/commands/applicant/AddApplicant.java]: Ok changed

**Review by Justinnje — APPROVED** [review]
LGTM

**Review by e0543517 — APPROVED** [review]
LGTM

**Pull request merged**